packages/flutter/lib/src/material/dialog.dart

```diff
@@ -507,6 +551,18 @@ class SimpleDialog extends StatelessWidget {
   /// the top padding ends up being 24 pixels.
   final EdgeInsetsGeometry contentPadding;
 
+  /// {@macro flutter.material.dialog.backgroundColor}
+  final Color backgroundColor;
+
+  /// The z-coordinate of this [Dialog].
+  ///
+  /// If this is `null`, [DialogTheme.elevation] will be used, if
```

Comment:
As before. In fact, just referring to an `@template` defined by Dialog would be OK.

If null then [DialogTheme.elevation] is used, and if that's null then the dialog's elevation is 24.0.

Reply:
Fixed in macro.